### PR TITLE
add alchemyjs v0.4.2 

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/GraphAlchemist/Alchemy
**Watch 25
  Star 185
 Fork 61**
@Amomo  could u help me check this PR for #6402 ? thank you!

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/alchemyjs/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/GraphAlchemist/Alchemy.git",
+    "basePath": "dist",
+    "files": [
+      "styles/images/*.png",
+      "alchemy.css",
+      "alchemy.min.css",
+      "alchemy.js"
+    ]
```
Comment (PeterDaveHello) [inline ajax/libs/alchemyjs/package.json]: Why not `"**/*"`?

**Comment by maruilian11** [comment]
@PeterDaveHello i thought verdor.js is used for test because it's under script folder, and files under fonts folder is owned by another lib.

**Comment by PeterDaveHello** [comment]
Under scripts folder means for test? Never know that.

**Comment by maruilian11** [comment]
ok, i will ask the lib owner.

**Comment by maruilian11** [comment]
@Amomo i have modified the basepath to `**/*`, could you help me check the PR? thank you!

**Comment by Amomo** [comment]
@maruilian11 
Do we need to host files in `"dist/styles/fonts"`?
We already have [font-awesome](https://cdnjs.com/libraries/font-awesome) in cdnjs.

And, according to the [document](http://graphalchemist.github.io/Alchemy/#/doc),
`"alchemy.js"`, `"alchemy.min.js"`, `"alchemy.css"`, and `"alchemy.min.css"` are main files for CDN.
"vendor.js" and "vendor.css" are used to load the dependencies.
Maybe we could also host "vendor.js" and "vendor.css" also.

In short, I think we could remove the fonts and keep others.

**Comment by maruilian11** [comment]
@PeterDaveHello i used to not adding `font` and `vendor*`, do we still need to use `**/*` in this case?

**Comment by PeterDaveHello** [comment]
It depends on if these fonts files are referenced by the js/css codes, please confirm that.

**Comment by PeterDaveHello** [comment]
Since vendor.css referenced them, so of course we should add it.

File: ajax/libs/alchemyjs/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "alchemyjs",
+  "version": "0.4.2",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/graphalchemist/alchemy.git"
```
Comment (PeterDaveHello) [inline ajax/libs/alchemyjs/package.json]: Url is case sensitive, and I would like to use https instead of http, so it should be `"https://github.com/GraphAlchemist/Alchemy.git"`.

Comment (maruilian11) [inline ajax/libs/alchemyjs/package.json]: got it
